src/implementations/vanilla/components/date-block/date-block.scss

```diff
@@ -0,0 +1,186 @@
+/*
+ * Date block
+ * @define date-block
+ */
+
+// Import base
+@use "sass:map";
+@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/spacing';
+@use '@ecl/theme-dev/maps/typography';
+@use '@ecl/layout-grid/mixins/breakpoints';
+
+$background-default: map.get(color.$color, 'blue');
```

Comment:
I didn't notice it before, but we are using a new naming convention for local variables defined in the components: it should start with an underscore (`$_background-default`)
I think it is part of the migration document